src/handlers/shared.ts

```diff
@@ -1,6 +1,7 @@
 import { getBotContext } from "../bindings";
 import { ActionHandler } from "../types";
 
+export const deadLinePrefix = "The time limit for this bounty is on";
```

Comment:
Where does this string manifest for the user?

Reply:
Once a hunter gets assigned by the bounty-admin, the message will be posted on the issue.